complaintdatabase/templates/landing-page.html

```diff
@@ -5,18 +5,21 @@
 {% block title %}Consumer Complaint Database{% endblock %}
 
 {% block content %}
-
 <div id="ccdb-landing">
   <!--
     If the data has not been updated, add class 'show-data-notification' to cf-notification
     If narratives have not been updated, add class 'show-narratives-notification'-->
-  <div class="cf-notification cf-notification__warning {% if data_down %} show-data-notification {% elif narratives_down %} show-narratives-notification {% endif %} ">
+    <div class="cf-notification cf-notification__warning {% if data_down %} show-data-notification {% elif narratives_down %} show-narratives-notification  {% endif %}">
```

Comment:
It''s a pre-existing variable that I'm re-using (it's also the variable that will be set if the data actually *is* down)